front/src/AddLogo.js

```diff
@@ -5,7 +5,7 @@ function AddLogo() {
     <div>
       <header>
         <h1>this is an h1 tag</h1>
-        <img src={logo}  alt="logo" />
```

Comment:
good job, please add the url of the logo (svg is recommended for higher resolutions)